app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="reg-container">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: {class: 'form-control'}) do |f| %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %><br />
+      <%= f.password_field :password, autocomplete: "current-password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit "Log in", class:'btn btn-block'%>
+    </div>
+  <% end %>
+
```

Comment:
![image](https://github.com/christianonoh/beyond-borders/assets/93039359/26e071b8-c31a-4606-b410-9fb289ae437f)

- [ ] Great work so far 👍 but it would be nice if you put placeholders in the input fields instead of using the labels 🤔 to make your design look similar to the given wireframe 😊